src/clients/HubPoolClient.ts

```diff
@@ -690,33 +751,24 @@ export class HubPoolClient extends BaseAbstractClient {
         continue;
```

Comment:
I think you also want to delete the entire block of code above this line.

![2023-07-19-201258_676x351_scrot](https://github.com/across-protocol/sdk-v2/assets/108695806/9f68c6b6-19cb-4dc5-835c-7199b2b14216)